src/vivarium_gates_child_iv_iron/data/lbwsg_paf_calculator.py

```diff
@@ -0,0 +1,134 @@
+from pathlib import Path
+import pickle
+import sys
+
+from loguru import logger
+import numpy as np
+import pandas as pd
+
+from vivarium import Artifact, InteractiveContext
+from vivarium_cluster_tools.utilities import mkdir
+from vivarium_gbd_access import gbd
+
+from vivarium_gates_child_iv_iron.constants import data_keys, metadata
+
+
+def get_relative_risks(config: Path, input_draw: int, random_seed: int, age_group_id: int) -> pd.DataFrame:
+    logger.remove()
+    sim = InteractiveContext(config, setup=False)
+
+    artifact_path = sim.configuration.input_data.artifact_path
+    artifact = Artifact(artifact_path)
+
+    # Make mapper for age_group_ids
+    gbd_age_bins = gbd.get_age_bins()
+    name_to_id_mapper = dict(zip(gbd_age_bins.age_group_name, gbd_age_bins.age_group_id))
+
+    age_bins = artifact.load(data_keys.POPULATION.AGE_BINS)
+    age_group_ids = age_bins.reset_index()["age_group_name"].map(name_to_id_mapper)
+    age_group_ids.index = age_bins.index
+
+    age_bins["age_group_id"] = age_group_ids
+    age_bins = age_bins.reset_index().set_index('age_group_id')
```

Comment:
Maybe a bit of a nitpick, but we perform the exact same transformation to the age_bins artifact twice in this file in different functions to add age group ID information. Would it be worth defining a new function to do this to avoid duplicating code?